shimmer/docs/iota-sdk/docs/how-tos/exchange-guide.mdx

```diff
@@ -43,63 +28,175 @@ Features of the IOTA SDK:
+  [`send`](../references/nodejs/classes/Account/#send).
+- The callbacks to `listen` now provide a properly typed
+  [`Event`](../references/nodejs/classes/Event/).
+
+</TabItem>
+<TabItem value="python" label="Python">
+
+- The pip package changes from `iota-wallet` to [`iota-sdk`](https://pypi.org/project/iota-sdk/).
+- [`wallet.generate_mnemonic()`](https://wiki.iota.org/shimmer/wallet.rs/references/python/iota_wallet/wallet/#generate_mnemonic)
+  becomes
+  [`utils.generate_mnemonic()`](../references/python/iota_sdk/utils/#generate_mnemonic).
+- [`wallet.generate_address()`](https://wiki.iota.org/shimmer/wallet.rs/references/python/iota_wallet/wallet/#generate_address)
+  becomes
+  [`wallet.generate_ed25519_addresses()`](../references/python/iota_sdk/wallet/#generate_ed25519_address).
+- [`account.send_amount`](https://wiki.iota.org/shimmer/wallet.rs/references/python/iota_wallet/account/#send_amount)
+  becomes
+  [`account.send`](../references/python/iota_sdk/wallet/account/#send).
+- The callbacks to `listen` now provide a properly typed
+  [`event`](../references/python/iota_sdk/types/event/).
+
+</TabItem>
+</Tabs>
 
 </details>
 
 ### 1. Set Up the IOTA SDK
 
-First, you should install the components needed to use the IOTA SDK and the binding of your choice; it may vary a bit from language to language. In case of the [NodeJs binding](../getting-started/nodejs.mdx), it is straightforward since it is distributed via the `npm` package manager.
+#### Install the IOTA SDK
+
+First, you should install the components needed to use the IOTA SDK and the binding of your choice; it may vary a bit
+from language to language.
 
 You can read more about [backup and security in this guide](https://wiki.iota.org/shimmer/introduction/how_tos/backup_security).
 
+<Tabs groupId="language" queryString>
+
+<TabItem value="rust" label="Rust">
+
+Check out our [Getting Started with Rust](../getting-started/rust.mdx) guide for more detailed instructions.
+
+<Tabs>
+<TabItem value="cargo" label="Cargo add">
+
+To start using the IOTA SDK in your Rust project, you can include it as a dependencies in your project's
+`Cargo.toml` by running:
+
 ```bash
-npm install @iota/sdk dotenv
+cargo add iota-sdk
+```
+
+</TabItem>
+
+<TabItem value="manual" label="Cargo.toml">
+
+To start using the IOTA SDK in your Rust project, you can include the following line under
+`dependencies` in your `Cargo.toml`:
+
+```toml
+[dependencies]
+iota-sdk = "1.0.0"
+```
+
+</TabItem>
+</Tabs>
+
+</TabItem>
+<TabItem value="typescript-node" label="Typescript (Node.js)">
+
+Check out our [Getting Started with Node.js](../getting-started/nodejs.mdx) guide for more detailed
+instructions.
+
+<Tabs groupId="nodePackageManager" queryString>
+<TabItem value="npm" label="npm">
+
+```sh
+npm i @iota/sdk dotenv
+```
+
+</TabItem>
+<TabItem value="yarn" label="Yarn">
+
+```sh
+yarn add @iota/sdk
```

Comment:
We could use npm2yarn instead 🤔 Wdyt?  https://docusaurus.io/docs/2.3.1/markdown-features/code-blocks#npm2yarn-remark-plugin